vpr/src/base/vpr_context.h

```diff
@@ -386,6 +386,11 @@ struct PlacementContext : public Context {
     ///@brief The pl_macros array stores all the placement macros (usually carry chains).
     std::vector<t_pl_macro> pl_macros;
 
+    ///@brief Stores ClusterBlockId of all movable clustered blocks.
+    std::vector<ClusterBlockId> movable_blocks;
```

Comment:
Can we remove the blocks_per_type data structures now? It seems these new structures replace them.

Reply:
I'd briefly define movable (not locked down to a single location).

Reply:
> Can we remove the blocks_per_type data structures now? It seems these new structures replace them.

Romoved blocks_per_type and the helper function to access it.



> I'd briefly define movable (not locked down to a single location).

Updated the comment